src/main/java/org/breedinginsight/brapi/v2/dao/BrAPIGermplasmDAO.java

```diff
@@ -145,21 +146,23 @@ private Map<String, BrAPIGermplasm> fetchProgramGermplasm(UUID programId) throws
                 api::searchGermplasmPost,
                 api::searchGermplasmSearchResultsDbIdGet,
                 germplasmSearch
-        ), program.getKey());
+        ), program);
     }
 
     /**
      * Process germplasm into a format for display
-     * @param programGermplasm
+     * @param germplasmList
      * @return Map<Key = string representing germplasm UUID, value = formatted BrAPIGermplasm>
      * @throws ApiException
      */
-    private Map<String,BrAPIGermplasm> processGermplasmForDisplay(List<BrAPIGermplasm> programGermplasm, String programKey) {
+    private Map<String,BrAPIGermplasm> processGermplasmForDisplay(List<BrAPIGermplasm> germplasmList, ProgramEntity program) throws ApiException {
+        String programKey = program.getKey();
         // Process the germplasm
         Map<String, BrAPIGermplasm> programGermplasmMap = new HashMap<>();
-        log.trace("processing germ for display: " + programGermplasm);
+        log.trace("processing germ for display: " + germplasmList);
         Map<String, BrAPIGermplasm> programGermplasmByFullName = new HashMap<>();
-        for (BrAPIGermplasm germplasm: programGermplasm) {
+        boolean isExpanded_programGermplasmByFullName = false;
```

Comment:
Unless the underscore in the name is part of a convention I don't know about, I would remove it.
`isExpandedProgramGermplasmByFullName` would be OK with me, something like `isMissingParents` would be OK too.